timeout/timeout.c

```diff
@@ -507,8 +507,10 @@ int parse_interval(const char *ss, struct TClock *interval) {
 		}
 	}
 
-	interval->sec = strlen(s) ? atoi(s) : 0;
-	interval->nsec = afterpoint ? atoi(afterpoint) : 0;
+	interval->sec = strlen(s)
+			? atoi(s) : 0;
```

Comment:
I don't think this indicates the if/else structure any clearer than the previous code. Maybe
```c
interval->sec = strlen(s) ? atoi(s)
                          : 0;
```
or
```c
interval->sec = strlen(s)
        ? atoi(s)
        : 0;
```
?

Reply:
Honestly, I don't remember doing this.
I'd go for the second option.

Reply:
Done: https://github.com/Projeto-Pindorama/heirloom-ng/pull/41/commits/ba133b5c6ca50a770ab2d7e23b6d57b32635d992